src/app/views/album-detail.html

```diff
@@ -1,14 +1,51 @@
-<div>
-    <h2>EN ESTA SECCION TIENE QUE APARECER LA TAPA DEL DISCO SELECCIONADO JUNTO CON LA LISTA DE TEMAS QUE TIENE EL DISCO.</h2>
-    <img src="https://i.scdn.co/image/66302ca80395b6be3600d5f0ef69db9e0c43f4f5" alt="">
-    <ul>
-        <li>Track 1</li>
-        <li>Track 2</li>
-        <li>Track 3</li>
-        <li>Track 4</li>
-        <li>Track 5</li>
-        <li>Track 6</li>
-        <li>Track 7</li>
-        <li>Track 8</li>
-    </ul>
+<div class="heading">
+    <a href="/#"><img class="header-icon" src="http://www.iconarchive.com/download/i85639/graphicloads/100-flat/zoom-search-2.ico" alt="Spotify search icon"></a>
```

Comment:
you can use font-awesome for a large but lightweight icon set and you can stylish them with css